packages/react-components/review/beta/react-components.api.md

```diff
@@ -1343,6 +1343,12 @@ export interface ImageGalleryStrings {
     downloadButtonLabel: string;
 }
 
+// @beta
+export interface InlineImage {
+    imgAttrs: React_2.ImgHTMLAttributes<HTMLImageElement>;
```

Comment:
Do we know why this is resolving to `React_2.....`?

Reply:
Not sure why React always resolve to React_2, this is everywhere in our codebase.